packages/workbox-build/src/entry-points/options/defaults.js

```diff
@@ -7,18 +7,21 @@
 */
 
 module.exports = {
+  babelPresetEnvTargets: ['chrome >= 56'],
   cleanupOutdatedCaches: false,
   clientsClaim: false,
   globFollow: true,
   globIgnores: ['**/node_modules/**/*'],
   globPatterns: ['**/*.{js,css,html}'],
   globStrict: true,
-  importWorkboxFrom: 'cdn',
-  injectionPointRegexp: /(precacheAndRoute\()\s*\[\s*\]\s*(\)|,)/,
+  injectionPoint: 'self.__WB_PRECACHE_MANIFEST',
```

Comment:
Sorry to keep bikeshedding this, but after playing with this locally a bit and seeing all the other uses of the term "manifest" without any mention of "precache", I think it'd be fine to keep this `self.__WB_MANIFEST` if that's your preference.

Reply:
Okay, switching back.